pkg/kubelet/cm/devicemanager/manager_test.go

```diff
@@ -1850,3 +1850,84 @@ func TestGetTopologyHintsWithUpdates(t *testing.T) {
 		})
 	}
 }
+func TestUpdateAllocatedResourcesStatus(t *testing.T) {
```

Comment:
https://github.com/kubernetes/kubernetes/issues/126357

This looks to cause some flakes.